src/main/java/uk/gov/hmcts/reform/blobrouter/reconciliation/model/in/Report.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.blobrouter.reconciliation.model.in;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class Report {
+    public final List<Envelope> envelopes;
+
+    public Report(@JsonProperty("envelopes") List<Envelope> envelopes) {
+        this.envelopes = envelopes;
+    }
+}
```

Comment:
this may get confusing... 
we call this a SupplierStatement in db, report is the thing we generate later on.

Reply:
I'll rename